charts/kor/values.yaml

```diff
@@ -0,0 +1,26 @@
+namespace: kor
+
+cronJob:
+  enabled: true
+  name: kor
+  schedule: "0 1 * * 1"
+  image:
+    repository: yonahdissen/kor
+    tag: latest
+  # e.g. kor configmap --include-namespace default,other-ns
+  korCommand: kor all
```

Comment:
Lets change this to command

Reply:
My idea was to have it configurable so if someone doesn't want to run `kor all` and instead run only `kor secrets` for example they could configure it through helm cli. e.g.

```
helm upgrade -i kor \
    --namespace kor \
    --create-namespace \
    --set cronJob.slackChannel=<slack-channel> \
    --set cronJob.slackToken=<slack-token> \
    --set cronJob.korCommand="kor secrets --include-namespace example-ns"
    ./charts/kor
``` 

If you still want it, I can hard code it into the CronJob definition as `kor all`. Let me know what you think.

Reply:
I just mean the variable name

Reply:
Oh, I completely misunderstood it 🤦